esp-storage/src/esp32.rs

```diff
@@ -34,23 +25,17 @@ const SPI_WRSR_2B: u32 = 1 << 22;
 const FLASH_CHIP_ADDR: u32 = 0x3ffae270;
 const FLASH_DUMMY_LEN_PLUS_ADDR: u32 = 0x3ffae290;
 
-#[inline(always)]
-#[link_section = ".rwtext"]
-pub(crate) fn cache_flush_rom(cpu_num: u32) {
-    unsafe {
-        let cache_flush_rom: unsafe extern "C" fn(u32) = core::mem::transmute(CACHE_FLUSH_ROM);
-        cache_flush_rom(cpu_num)
-    }
-}
-
-#[inline(always)]
-#[link_section = ".rwtext"]
-pub(crate) fn cache_read_enable_rom(cpu_num: u32) {
-    unsafe {
-        let cache_read_enable_rom: unsafe extern "C" fn(u32) =
-            core::mem::transmute(CACHE_READ_ENABLE_ROM);
-        cache_read_enable_rom(cpu_num)
-    }
+crate::rom_fn! {
+    fn esp_rom_cache_flush(cpu_num: u32) = 0x40009a14,
```

Comment:
super trivial/optional comment, semi colon is a bit more natural here.
```suggestion
    fn esp_rom_cache_flush(cpu_num: u32) = 0x40009a14;
```
I'm no good at macros so idk if it's possible

Reply:
Of course it's possible. I originally went with , because it's optional for the last row, but that's also something not very natural I guess.